Sources/HttpPipelineTestSupport/HttpPipelineTestSupport.swift

```diff
@@ -38,17 +38,11 @@ extension Response: Snapshot {
 
 extension Conn: Snapshot {
   public var snapshotFormat: String {
```

Comment:
Cleaning up the snapshot format a bit to make it look more like a raw request/response.